### PR TITLE
Update contributors for Third Sector Design

### PR DESCRIPTION
**Pull request by wmortada**
Update the contributors for Third Sector Design

### CODE REVIEW

**Comment by demeritcowboy** [comment]
I thought the contributors.txt file was an archive (see line 1) but there do seem to be some recent commits so ok.

**Comment by wmortada** [comment]
I wasn't sure, but as you say it seems to have been updated by others.

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -101,7 +101,7 @@ Systopia - Björn Endres, Thomas Leichtfuss
 Tadpole Collective - Kevin Cristiano
 Tanya Bouman
 Team Expansion - Greg Harris
-Third Sector Design - Michael McAndrew
+Third Sector Design - Michael McAndrew, Kurund, Jalmi, William Mortada
```
Comment (mlutfy) [inline CONTRIBUTORS.txt:104]: There's a typo: can you remove the comma between Kurund and Jalmi?

Comment (wmortada) [reply CONTRIBUTORS.txt]: Sorry! Fixed it now.

**Comment by joshgowans** [comment]
The contributors.txt file _should_ be an archive and we _should_ have a more public listing of contributors on civicrm.org. Keywords are highlight. :D The listing is a work in progress currently as we're rebuilding it and the partner listing using SearchKit.

**Pull request merged**